src/main/java/com/opencart/steps/AccountDashboardBL.java

```diff
@@ -0,0 +1,153 @@
+        clickOnContinueButton();
+        return this;
+    }
+
+    /**
+     * Method inputs new First Name data to the First Name field on the Edit Account page.
+     */
+    private void inputNewFirstName(String newFirstName) {
+        accountDashboard.getFirstNameInput().clear();
+        accountDashboard.getFirstNameInput().sendKeys(newFirstName);
+    }
+
+    /**
+     * Method inputs Email data to the First Name field on the Edit Account page.
+     *
+     * @param email is Email data
+     */
+    private void inputEmail(String email) {
+        accountDashboard.getEmailInput().clear();
+        accountDashboard.getEmailInput().sendKeys(email);
+    }
+
+    /**
+     * Method inputs new Password data to the Password and Confirm Password fields
+     * on the Edit Account page.
+     *
+     * @param password is Password data
+     */
+    private void inputPassword(String password) {
+        accountDashboard.getPasswordChangeInput().clear();
+        accountDashboard.getPasswordChangeInput().sendKeys(password);
+        accountDashboard.getPasswordConfirmInput().clear();
+        accountDashboard.getPasswordConfirmInput().sendKeys(password);
+    }
+
+    /**
+     * Method clicks on Continue Button on the Edit Account page.
+     */
+    private void clickOnContinueButton() {
+        accountDashboard.getContinueButton().click();
+    }
+
+    /**
+     * Method clicks on Edit Password Button on the Edit Account dashboard page.
+     */
+    public AccountDashboardBL clickOnEditPasswordButton() {
+        accountDashboard.getEditPasswordButton().click();
+        return this;
+    }
+
+    /**
+     * Method clicks on Edit Account Button on the Edit Account dashboard page.
+     *
+     * @return AccountDashboardBL
+     */
+    public AccountDashboardBL clickOnEditAccountButton() {
+        accountDashboard.getEditAccountButton().click();
+        return this;
+    }
+
+    /**
+     * Method verifies if Success message appears after User Account was edited
+     * and Continue Button was clicked.
+     *
+     * @return AccountDashboardBL
+     */
+    public AccountDashboardBL verifyAccountEdit() {
+        String expectedMessage = "Success: Your account has been successfully updated.";
+        Assert.assertEquals(accountDashboard.getSuccessMessage().getText(), expectedMessage, "\nSuccess message after editing Account information did not appear.");
+        return this;
+    }
+
+    /**
+     * Method verifies if Success message appears after User Password was edited
+     * and Continue Button was clicked.
+     *
+     * @return AccountDashboardBL
+     */
+    public AccountDashboardBL verifyPasswordEdit() {
+        String expectedMessage = "Success: Your password has been successfully updated.";
+        Assert.assertEquals(accountDashboard.getSuccessMessage().getText(), expectedMessage, "\nSuccess message after editing Account information did not appear.");
+        return this;
+    }
```

Comment:
Were you thinking about additional verification? For example logout, after changing the password and try to login with a new one. I would recommend to implement it too.

Reply:
I have added test "changePasswordVerifyLogin ()"
<img width="746" alt="Screen Shot 2020-11-19 at 13 47 15" src="https://user-images.githubusercontent.com/73054470/99662300-bcaa2080-2a6d-11eb-9f51-a6937d8f6b67.png">